icon-loader.js

```diff
@@ -9,6 +9,8 @@ module.exports = function shopifyIconLoader(source) {
     return fill.includes('#FFF') ? 'fill="currentColor"' : '';
   });
 
+  // Issue with ESLint recognizing this as needing an object destructure
+  // eslint-disable-next-line prefer-destructuring
```

Comment:
It's related to the [`enforceForRenamedProperties`](https://github.com/Shopify/eslint-plugin-shopify/blob/master/lib/config/rules/ecmascript-6.js#L43) we have enabled. 

I think what's happening is the rule is expecting something like
```
const [, viewBox] = VIEWBOX_REGEX.exec(finalSource);
```

See [docs](http://eslint.org/docs/rules/prefer-destructuring#options)